spec/index.bs

```diff
@@ -2354,6 +2359,9 @@ enum GPUCompareFunction {
         1. Set |s|.{{GPUSampler/[[descriptor]]}} to |descriptor|.
         1. Set |s|.{{GPUSampler/[[compareEnable]]}} to false if the {{GPUSamplerDescriptor/compare}} attribute
                 of |s|.{{GPUSampler/[[descriptor]]}} is null or undefined. Otherwise, set it to true.
+        1. Set |s|.{{GPUSampler/[[filterEnable]]}} to false if all of {{GPUSamplerDescriptor/magFilter}},
+            {{GPUSamplerDescriptor/minFilter}}, and {{GPUSamplerDescriptor/mipmapFilter}}, are {{GPUFilterMode/"nearest"}},
+            of |s|.{{GPUSampler/[[descriptor]]}} is null or undefined. Otherwise, set it to true.
```

Comment:
```suggestion
            {{GPUSamplerDescriptor/minFilter}}, and {{GPUSamplerDescriptor/mipmapFilter}}, of |s|.{{GPUSampler/[[descriptor]]}}
            are {{GPUFilterMode/"nearest"}}, null or undefined. Otherwise, set it to true.
```